### ISSUE TITLE
Custom UI color on default theme

### ISSUE THREAD

**Issue by rsolvang** [espocrm/espocrm#122]
It would be nice to be able to define a custom color for the default theme. Since the interface is so simple and well design with only one main color, this could be relatively easy to implement. 

In most cases, a company have one color they associate with their profile, which makes it very easy to brand the UI and make it more familiar. It shouldn't be necessary to make a new theme just to choose a different color.

**Comment by yurikuzn** [comment]
Not very easy. We use bootstrap and generate css from less files to build themes. Colors are stored in varables and they are altered (shaded) during build process.

**Comment by yurikuzn** [comment]
It's not difficult to write theme builder and host it in a web. As an idea for a community.

**Comment by rsolvang** [comment]
I'm not very familiar with the build process of this project, but I would imagine changing the main color in the LESS file, then rebuilding the CSS, could get the job done – if all other colors (like shaded etc) is connected to the main color. A theme builder would also be a nice feature, but I think most businesses would get their needs met by changing the logo and the main color. This is something everyone can do, and it is the shortest way to satisfaction for the user. The easier, the better.

The beauty of this CRM is it's simplicity and it's non-cluttered UI. This strength could be compromised by custom themes that brake the conventions by changing more than the colors, increasing the chances for broken functionality when new updates are released.

Anyway, this is only nitpicking, but it was something I found myself wanting when I set up the installation.
